modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -286,7 +286,23 @@ Refer to https://help.mulesoft.com/s/article/How-to-Troubleshoot-RTF-Appliance-U
 
 == Roll Back an Appliance Upgrade
 
-If you cannot resolve issues with an upgrade, you can roll back the upgrade steps that executed.
+If you cannot resolve issues with an upgrade, you can roll back the upgrade using one of the following options:
+
+* Automatic rollback with a single command
+* Manual rollback
+
+=== Automatic Rollback With a Single Command
+
+Enter the following command:
+```
+rtfctl appliance rollback
+```
+[NOTE]
+This option is only supported for newer appliance versions. If the command output shows that the `rtfctl appliance rollback` command is not supported, then perform a manual rollback.
+
+=== Manual Rollback
+
+To perform a manual rollback of the upgrade steps that executed:
```

Comment:
"of the upgrade steps that executed": perhaps "of a partially completed upgrade"?